### ISSUE TITLE
Evernote link / documentation broken?

### ISSUE THREAD

**Issue by horsto** [cleterrier/Process_Images#1]
Hey Christophe, 
is the evernote link broken? Is there a documentation available somewhere? 
Thanks!

**Comment by cleterrier** [comment]
Hi,

Don't know what happened - thanks for telling me. The link in the readme should work now.

**Comment by horsto** [comment]
Thanks a lot! works! 
One additional question: In the install instructions it says "2. Download the Process_Images.ijm toolset, place it in the /macros/toolsets/ Fiji folder.". Maybe I'm just blind, but where is that .ijm? 
(However: The macros work without it as well - thanks for uploading them! Very useful!)

**Comment by cleterrier** [comment]
You're right, it wasn't in the repo (just in the Fiij update site). Should be here now.
